app/src/main/res/values/button_themes.xml

```diff
@@ -9,9 +9,15 @@
     <style name="Hedvig.Button.Large">
         <item name="android:padding">@dimen/base_margin_double</item>
         <item name="shapeAppearance">?shapeAppearanceLargeComponent</item>
+        <item name="colorButton">@color/colorLargeButton</item>
```

Comment:
I think these colors should be the same for all filled buttons

Reply:
Maybe we should talk to Zak about it?

Reply:
Yes